lib/generators/solidus_editor/install/install_generator.rb

```diff
@@ -3,19 +3,36 @@
 module SolidusEditor
   module Generators
     class InstallGenerator < Rails::Generators::Base
-      source_root File.expand_path('../../../templates', __dir__)
+      source_root File.expand_path('templates', __dir__)
+      class_option :auto_run_migrations, type: :boolean, default: false
+
+      def add_javascripts
+        append_file 'vendor/assets/javascripts/spree/backend/all.js', "//= require spree/backend/solidus_editor\n"
+
+        if Sprockets::VERSION.to_i >= 4 && Sprockets::VERSION.to_i < 5
```

Comment:
```ruby
sprockets_version = Gem::Version.new(Sprockets::VERSION) 
if sprockets_version >= Gem::Version.new('4') && sprockets_version <= Gem::Version.new('5')
```
This is much better, since it will run a comparison on the actual version number, whereas `to_i` may yield weird results.